Tests/js/views/tabs/ez-locationviewdetailstabview.html

```diff
@@ -10,6 +10,12 @@
 
 <script type="text/x-handlebars-template" id="locationviewdetailstabview-ez-template">
     <button class="ez-asynchronousview-retry">Retry</button>
+    <select class="ez-subitems-ordering-sort-type" >
+        <option selected value="PRIORITY">Priority</option>
```

Comment:
you should have several options in the select so that simulating the change is closer to the real use case.